vector/src/main/java/im/vector/app/features/home/room/detail/timeline/item/MessageTextItem.kt

```diff
@@ -87,24 +87,22 @@ abstract class MessageTextItem : AbsMessageItem<MessageTextItem.Holder>() {
                 it.bind(holder.messageView)
             }
         }
-        val textFuture = if (bindingOptions?.canUseTextFuture.orFalse()) {
-            PrecomputedTextCompat.getTextFuture(
-                    message ?: "",
-                    TextViewCompat.getTextMetricsParams(holder.messageView),
-                    null)
-        } else {
-            null
-        }
         super.bind(holder)
         holder.messageView.movementMethod = movementMethod
         renderSendState(holder.messageView, holder.messageView)
         holder.messageView.onClick(attributes.itemClickListener)
         holder.messageView.onLongClickIgnoringLinks(attributes.itemLongClickListener)
 
+        message?.let { holder.messageView.setTextWithEmojiSupport(it, bindingOptions) }
```

Comment:
I guess that `message` should never be null, but the change is not strictly equivalent to what we had before: `message ?: ""`.

Maybe continue to fallback to empty string, to ensure that the text is always updated, or change `message` to a `lateinit CharSequence`

(if message is null, the TextView content will not be reset and will display the previous bound value)

Reply:
ah good catch, resetting to null seems safer, I don't know the epoxy lifecycle well enough to trust using a late init 

b5ce8dcb3219edb11d7a5055759e3124fa8894fe